packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -103,39 +103,48 @@ export const TranslationSidebar = forwardRef<
         );
       }
     }, [word.id, notesQuery]);
+    const saveTranslatorNoteMutation = useMutation({
+      mutationFn: async (noteContent: string) =>
+        await apiClient.words.updateTranslatorNote({
+          wordId: word.id,
+          language,
+          note: noteContent,
+        }),
```

Comment:
I don't think the mutation function should capture variables from outside it's scope. We should make the argument to the mutation be an object of `wordId`, `language`, and `note`, and then pass those in when executing

Reply:
Done.